### PR TITLE
Create Sign in

### PR DESCRIPTION
**Pull request by udyogi-siphara**
[ITP-1758]

[ITP-1758]: https://fcodelabs.atlassian.net/browse/ITP-1758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/SignIn.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    
+    //add the data using array for useSate 
+    const [names,setNames] = useState(['sip','usra','udy','baji']);
+    // for select a name in array
+    const [selectedName,setSelectedName] = useState('');
+
+    // this event handler use for random btn
+    const handleClick = () => {
+        //get the indexnum in the 'names' array
+        const randomIndex = Math.floor(Math.random() * names.length);
```
Comment (shamal-iroshan) [inline src/components/SignIn.tsx]: move this random name-generating function to a util file inside a folder named utility

File: src/components/SignIn.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        const randomName = names[randomIndex];
+        //set randomNames to state
+        setSelectedName(randomName);
+        
+    }
+
+    
+    // =========================Styles start===================
+
+    const paperStyle={
```
Comment (shamal-iroshan) [inline src/components/SignIn.tsx]: don't use styles like this with mui, use sx prop or styled to create styles when using mui

**Review by shamal-iroshan — CHANGES_REQUESTED** [review]
fix the commented things

File: src/components/SignIn.tsx (hunk 2 above)
Comment (udyogi-siphara) [reply src/components/SignIn.tsx]: Updated

File: src/components/SignIn.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    //useNavigate is equle to navigate variable
+    const navigate = useNavigate();
+
+    //navigate to home using a function
+    const navigateToHome = () =>{
+        navigate('/home');
+    }
+    
+    //add the data using array for useSate 
+    const [names,setNames] = useState(['sip','usra','udy','baji']);
```
Comment (udaraJ08) [inline src/components/SignIn.tsx:19]: Use a different random name generating approach

**Review by shamal-iroshan — APPROVED** [review]

**Pull request merged**